content/reference/sql/version-control/remotes.md

```diff
@@ -0,0 +1,115 @@
+
+## DoltLab
+
+[DoltLab](https://www.doltlab.com) is a version of [DoltHub](https://www.dolthub.com) you can deploy in your own network. It looks very similar to DoltHub. See the [DoltLab Guide](../../../guides/doltlab/installation.md) if you are interested in using DoltLab as a remote.
+
+## Filesystem
+
+Filesystem based remotes allow you to push/pull data from any location that can be accessed via the filesystem. This may be a directory on your local disk, or any other storage location that can be mounted to the filesystem. To add afilesystem based remote you need a URL with the `file://` protocol.
+
+**Linux / OSX Examples**
+
+ * Adding a remote 
+```
+dolt remote add origin file:///Users/brian/datasets/menus  
+```
+ * Cloning
+```
+dolt clone file:///Users/brian/datasets/menus
+```
+
+**Windows Examples**
+
+ * Adding a remote
+```
+dolt remote add origin file:///c:/Users/brian/datasets/menus
+```
+ * Cloning 
+```
+dolt clone file:///c:/Users/brian/datasets/menus
+```
+
+It's important to note that a directory-based remote is not the same as a workspace for a dolt clone, and the directory listed above as a remote file URL is not a dolt repository created or cloned with the [Dolt](https://doltdb.com) cli. Similarly, a [Dolt](https://doltdb.com) repository directory's file URL [cannot be used as a remote directly](https://github.com/dolthub/dolt/issues/1860).
+
+## AWS
+
+AWS remotes use a combination of Dynamo DB and S3. The Dynamo table can be created with any name but must have a primary
+key with the name "db".
+
+![Create a Dynamo Table with a primary key of: db](../images/create_dynamo_table.png)
+
+This single dynamo table can be used for multiple unrelated remote repositories.  Once you have a Dynamo DB table, and an S3 bucket setup you can add an aws remote using a URL with the protocol `aws://`. To add a remote named "origin" to my "menus" repository using an S3 bucket named `dolt_remotes_s3_storage` and a Dynamo DB table named `dolt_dynamo_table` you would run:
+
+```
+dolt remote add origin aws://[dolt_dynamo_table:dolt_remotes_s3_storage]/menus
+```
+
+This same URL can then be used to clone this database by another user.
+
+```
+dolt clone aws://[dolt_remotes:dolt_remotes_storage]/menus
+```
+
+In order to initialize your system to be able to connect to your AWS cloud resources see [Amazon's documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) on configuring your credential file. [Dolt](https://doltdb.com) also provides additional parameters you may need to provide when adding an AWS remote such as `aws-creds-profile`, and `aws-region`.`aws-creds-profile` allows you to select a profile from your credential file. If it is not provided then the default profile is used. `aws-region` allows you to specify the region in which your Dynamo DB table and S3 bucket are located. If not provided it will use the default region from the current profile.
```

Comment:
```suggestion
In order to initialize your system to be able to connect to your AWS cloud resources see [Amazon's documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) on configuring your credential file. [Dolt](https://doltdb.com) also provides additional parameters you may need to provide when adding an AWS remote such as `aws-creds-profile`, and `aws-region`.`aws-creds-profile` allows you to select a profile from your credential file. If it is not provided then the default profile is used. `aws-region` allows you to specify the region in which your DynamoDB table and S3 bucket are located. If not provided, it will use the default region from the current profile.
```